app/views/ads/_ad.html.erb

```diff
@@ -30,16 +30,16 @@
           </div>
           <% if ad.maps_query.present? %>
             <div class="column is-narrow">
-              <div class="tags has-addons">
-                <span class="tag">Lokacija na karti</span>
-                <span class="tag">
-                  <a href="<%= maps_url(ad.maps_query) %>" target="_blank">
+              <a href="<%= maps_url(ad.maps_query) %>" target="_blank" style="display: block">
```

Comment:
We should avoid inline styles.
Please use `is-block` CSS class that comes with Bulma.
See: https://bulma.io/documentation/helpers/visibility-helpers/